tests/test_cli.py

```diff
@@ -668,3 +670,73 @@ def mock_operation(*args, **kwargs) -> int:
     
     assert exp_output in captured.out
     assert actual_retval == exp_retval
+
+
+def _good_build(*args, **kwargs):
+    print("LGTM")
+
+
+def _bad_build(*args, **kwargs):
+    raise Exception
+
+
+@contextmanager
+def _totally_real_temp_dir(*a, **kw):
```

Comment:
OK, so, I don't want to be the killjoy here, but as this is code that _might_ be read by users, I'd use more explanatory names for functions and variables, and I'd apply the same standard to mock values. I know, I know, I'm old.

Reply:
Hahaha, tbf this is absolutely a name I will regent the literally the next time I have to edit this test. Consider it changed to something reasonable!